test/cmdlineTests.sh

```diff
@@ -71,6 +71,15 @@ function update_expectation {
     local newExpectation="${1}"
     local expectationFile="${2}"
 
+    if [[ $newExpectation == "" || $newExpectation -eq 0 ]]
+    then
+    	if [[ -f $expectationFile ]]
+    	then
```

Comment:
Looks like you're mixing tabs and spaces here. Note that the file uses spaces for indentation.